fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -399,6 +396,36 @@ public void testCreateVersionDuplicateMementoDatetime() throws Exception {
         }
     }
 
+    @Test
+    public void testDeleteAndPostContainerMemento() throws Exception {
+        createVersionedContainer(id);
+
+        final String mementoUri = createContainerMementoWithBody(subjectUri, MEMENTO_DATETIME);
+
+        assertEquals("Expected delete to succeed",
+                NO_CONTENT.getStatusCode(), getStatus(new HttpDelete(mementoUri)));
+
+        final String mementoTombstoneUri = mementoUri + "/fcr:tombstone";
+        assertEquals("Expected delete to succeed",
+                NO_CONTENT.getStatusCode(), getStatus(new HttpDelete(mementoTombstoneUri)));
+
+        assertEquals("Delete memento must be removed",
+                NOT_FOUND.getStatusCode(), getStatus(new HttpGet(mementoUri)));
+
+        final String recreatedUri = createContainerMementoWithBody(subjectUri, MEMENTO_DATETIME);
+        assertEquals("Recreated memento must exist",
+                OK.getStatusCode(), getStatus(new HttpGet(recreatedUri)));
+    }
+
+    @Test
+    public void testDeleteAndPostBinaryMemento() throws Exception {
```

Comment:
Would you mind either adding content to these empty tests, creating JIRAs for them, or removing them?